inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -372,6 +372,9 @@ def execute_from_manifest(self,
                 self._dispatcher_broker.send_result(SOTA_FAILURE)
                 raise SotaError(SOTA_FAILURE)
 
+    def _is_reboot_device(self) -> bool:
+        return False if self._reboot_device in ["No", "N", "n", "no", "NO"] else True
```

Comment:
The method `_is_reboot_device` could be simplified by directly returning the boolean expression instead of using an if-else statement. [medium]

Reply:
We do this in two places, so I'd rather have a separate private method to do the work.